src/app/camo-card/camo-card.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="card">
+  <div class="progress-number">
+    {{ progress[weapon.id].current }} /
```

Comment:
replace this with progress.current and progress.total. The reason is that we've passed in that data from home-page.component.html when we call the component. `<app-camo-card [weapon]='weapons' [progress]='progressData'></app-camo-card>`.

the @Input() is weapon and progress, not @Input() progressData.